fsharp-backend/src/LibBackend/PackageManager.fs

```diff
@@ -331,14 +333,20 @@ let allFunctions () : Task<List<PT.Package.Fn>> =
               function_ = fnname
               version = version }
           use _span =
-            LibService.Telemetry.child "decode package binary" [ "fn", string name ]
+            LibService.Telemetry.child
+              "decode package binary"
+              [ "fn",
+                name
+                |> PT2RT.FQFnName.PackageFnName.toRT
+                |> RT.FQFnName.PackageFnName.toString
+                :> obj ]
```

Comment:
I'm curious why this needs to be `:> obj`'d.

Reply:
`child` takes a `List<string * obj>` (which eventually makes its way into OpenTelemetry which uses objs.